spec/controllers/submissions_controller_spec.rb

```diff
@@ -123,6 +123,9 @@
 
     context 'submitting a url' do
       describe 'should add url files' do
+        before :each do
+          @assignment.update(url_submit: true)
```

Comment:
make all of these `update!` so that it doesn't fail silently if there is an issue setting the value